tx-parser/src/lib.rs

```diff
@@ -117,12 +117,12 @@ impl<I> Tx<I>
                                  // TODO: entid or special keyword :db/tx?
                                  optional(Tx::<&[Value]>::entid()),
```

Comment:
Drop this, it's not legal.  Maybe you did in another commit and I didn't see it?